### PR TITLE
feat(proxy): add startup sanity checks to make it easier to get this up and running

### PR DESCRIPTION
**Pull request by ondrej-til**

### CODE REVIEW

File: src/sanity/sanity-checks.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+      } catch (err) {
+        // no error except gRPC status 5 (i.e. NOT FOUND) is allowed
+        if (!err.code || err.code != 5) {
+          throw Error(`Unable to read a secret. See below for details.\n${err.message}`)
+        }
+      }
+    }, true),
+    check('API token is set', async () => {
+      const token = await appConfig.apiToken
+      if (!Boolean(token)) {
```
Comment (janjilek) [inline src/sanity/sanity-checks.ts:91]: You should check for the apiToken length. It should be at least 32 characters long

File: src/sanity/sanity-checks.ts [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+  printHeader('RUNNING CHECKS', '=')
+  let abort = false
+  let total = 0
+  let skipped = 0
+  let failed = 0
+  for (let suite of suites) {
+    printHeader(`${suite.name}`, '-')
+    for (let check of suite.checks) {
+      await check.fn().then(() => {
+        console.log(`• ${check.name} - \x1b[32mOK\x1b[0m`)
```
Comment (janjilek) [inline src/sanity/sanity-checks.ts:240]: Color output should be disabled by default, it's usually useless in the cloud enviroment.

File: src/sanity/sanity-checks.ts (hunk 1 above)
Comment (ondrej-til) [reply src/sanity/sanity-checks.ts:91]: fixed

File: src/sanity/sanity-checks.ts (hunk 2 above)
Comment (ondrej-til) [reply src/sanity/sanity-checks.ts]: removed

**Review by janjilek — APPROVED** [review]

**Pull request merged**